src/mgmt/srv_pool.c

```diff
@@ -479,12 +537,15 @@ ds_mgmt_create_pool(uuid_t pool_uuid, const char *group, char *tgt_dev,
 	if (rc) {
```

Comment:
or just declare rc_cleanup within this code block (our coding style allows this, but not arbitrary place)

Reply:
Fixed in #2860